python_docs_theme/theme.conf

```diff
@@ -4,8 +4,8 @@ stylesheet = pydoctheme.css
 pygments_style = sphinx
 
 [options]
-bodyfont = 'Lucida Grande', Arial, sans-serif
-headfont = 'Lucida Grande', Arial, sans-serif
+bodyfont = 'SourceSansProRegular','Lucida Grande', Arial, sans-serif
+headfont = 'SourceSansProRegular','Lucida Grande', Arial, sans-serif
```

Comment:
This is unrelated and should be discussed in a seperate issue

Reply:
OK, I will add this as a separate PR for discussion. Python.org uses this font, so I changed it because I think SourceSans looks more modern.